assets/stylesheets/bootstrap/mixins/_alerts.scss

```diff
@@ -15,4 +15,10 @@
     color: $text-color !important;
     text-decoration: underline;
   }
+  .btn-alert {
+    text-decoration: none;
+    &:hover {
+      color: $white !important;
+    }
+  }
```

Comment:
Overwrite the two rules immediately above when we have an `a` styled as a button inside an alert using `btn-alert`

Should look like 
![image](https://user-images.githubusercontent.com/5774647/94945501-ab637f80-04d2-11eb-90a8-0805b6eadbb4.png)
and on hover
![image](https://user-images.githubusercontent.com/5774647/94945519-b5857e00-04d2-11eb-9731-15b2517b143f.png)